README.md

```diff
@@ -158,43 +158,68 @@ Setting up and configuring of all these protections depends on and requires a lo
 
 ## Set up development environment
 
-### Local setup
+### Developing dependencies
+
+- API
+  - Required packages:
```

Comment:
I see that this list is a bit different from the install dependencies scripts from the previous PR (like, I don't see here `libheif` and `g++`, but see the `libexif`, missing in the script). Please double-check and let me know why is that)
I assume that there only the dependencies, needed to build the code, are listed, but since you created scripts to set the local dev env up, I think that there should be the possibility or an additional script to install the developing dependencies as well

Reply:
It's a typo.

I hope to show depending packages clearly in README. Also `scripts/apt/install_build_dependencies.sh` contains cleanup commands. I don't know if it's good to let developers run it directly.

For `homebrew` scripts, I may remove `scripts/homebrew/install_*_dependencies.sh` as the content are already in README. I'll keep `scripts/homebrew/set_compiler_env.sh` as it should be run every time before building API.

Reply:
I updated as I said. Let me know what you think.

Reply:
> `scripts/apt/install_build_dependencies.sh` contains cleanup commands. I don't know if it's good to let developers run it directly

That is a good catch! Definitely, not all of those commands should be executed on a dev env, as some of them are destructive (they are fine for an image, where we're sure there will be no future installations):
- `rm -rf /var/lib/apt/lists/*`
- `apt-get purge -y ${BUILD_DEPENDS[@]}`
These 2 commands should not be executed on a long-leaving environment, like a dev system or a server.

Reply:
> I may remove `scripts/homebrew/install_*_dependencies.sh` as the content are already in README

Readme - is a doc, while script helps to simplify the setup by running 1 command instead of many.

Reply:
I agree `README` is a doc. It should tell the reader details, but not an emergency playbook to operate blindly. Here, the content shows exactly what needs to be installed. If we add or remove any dependencies in the future, it's easy to see the change instead of in scripts.

Also, putting these commands into scripts won't reduce the running commands, if you don't want to install extra packages during building. It requires at least two scripts, build dependencies, and dev tools, because dev tools are not required during building.